frontend/packages/dev-console/src/components/project-access/ProjectAccess.tsx

```diff
@@ -95,7 +95,7 @@ const ProjectAccess: React.FC<ProjectAccessProps> = ({
           values: {
             projectAccess: values.projectAccess,
           },
-          status: { success: `Successfully updated the ${formName}.` },
+          status: { success: t('devconsole~Successfully updated the {{formName}}.', { formName }) },
```

Comment:
I would remove the variable here and drop the prop `formName` from ProjectAccess. ProjectAccessPage sets this always to "project access".

This would also make the translation much easier.